src/fides/api/alembic/migrations/versions/548a1ac26771_fideslang_3.0_remove_references.py

```diff
@@ -0,0 +1,244 @@
+"""remove data_qualifier
+
+Revision ID: 548a1ac26771
+Revises: 848a8f4125cf
+Create Date: 2023-12-08 22:07:38.617038
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "548a1ac26771"
+down_revision = "848a8f4125cf"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    ## Remove data qualifier references
+    op.drop_index("ix_ctl_data_qualifiers_fides_key", table_name="ctl_data_qualifiers")
+    op.drop_index("ix_ctl_data_qualifiers_id", table_name="ctl_data_qualifiers")
+    op.drop_table("ctl_data_qualifiers")
+    op.drop_column("ctl_datasets", "data_qualifier")
+    op.drop_index("ix_ctl_systems_name", table_name="ctl_systems")
+    op.drop_column("privacydeclaration", "data_qualifier")
+    op.drop_constraint("purpose_constraint", "tcf_purpose_overrides", type_="unique")
```

Comment:
ah this is my fault not properly defined on model, fixed in overhaul PR

Reply:
ah right i hadn't even noticed that here, shows you how closely i inspected the autogenerated migrations 😬 

i'll remove from here, thanks for catching!